backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -28,21 +27,16 @@
 import com.google.api.services.gmail.GmailScopes;
 import com.google.api.services.gmail.model.Message;
 import com.google.common.collect.ImmutableList;
-import java.io.ByteArrayOutputStream;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.security.GeneralSecurityException;
-import java.util.List;
-import java.util.Properties;
+import com.google.inject.Inject;
+import org.apache.commons.codec.binary.Base64;
 import javax.mail.Message.RecipientType;
 import javax.mail.MessagingException;
 import javax.mail.Session;
 import javax.mail.internet.InternetAddress;
 import javax.mail.internet.MimeMessage;
-import org.apache.commons.codec.binary.Base64;
+import java.io.*;
```

Comment:
Please use granular imports instead of * imports. Star imports can add a lot of unexpected classes to the namespace and obscure which classes are actually needed.

Reply:
Ack